msgid ""
msgstr ""
"Project-Id-Version: univention-management-console-module-appcenter\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2023-07-28 19:19+0200\n"
"PO-Revision-Date: 2020-09-15 14:46+0200\n"
"Last-Translator: Univention GmbH <packages@univention.de>\n"
"Language-Team: German <packages@univention.de>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: umc/python/appcenter/__init__.py:366
#, python-format
msgid "%d Apps: Connecting to %s"
msgstr "%d Apps: Verbinde zu %s"

#: umc/python/appcenter/__init__.py:313
#, python-format
msgid "%d Apps: Running tests"
msgstr "%d Apps: Starte Tests"

#: umc/python/appcenter/__init__.py:364
#, python-format
msgid "%s: Connecting to %s"
msgstr "%s: Verbinde zu %s"

#: umc/python/appcenter/__init__.py:311
#, python-format
msgid "%s: Running tests"
msgstr "%s: Starte Tests"

#: umc/python/appcenter/__init__.py:416
msgid ""
"A conflict between the system network settings and the docker bridge default "
"network has been detected."
msgstr ""
"Es wurde ein Konflikt zwischen den Netzwerkeinstellungen des Systems und der "
"Docker Bridge entdeckt."

#: umc/python/appcenter/__init__.py:555 umc/python/appcenter/__init__.py:695
msgid "Another package operation is in progress"
msgstr "Zur Zeit wird eine andere Paketverarbeitung durchgeführt"

#: umc/python/appcenter/sanitizers.py:57 umc/python/appcenter/sanitizers.py:61
#: umc/python/appcenter/sanitizers.py:65
msgid "Cannot connect to the LDAP service."
msgstr "Konnte nicht zum LDAP-Dienst verbinden."

#: umc/python/appcenter/sanitizers.py:80 umc/python/appcenter/__init__.py:468
#: umc/python/appcenter/__init__.py:489
#, python-format
msgid "Could not find an application for %s"
msgstr "Konnte keine Applikation für %s finden"

#: umc/python/appcenter/__init__.py:435
msgid "Could not load suggestions."
msgstr "Konnte App-Vorschläge nicht laden."

#: umc/python/appcenter/__init__.py:335
#, python-format
msgid "Installing %s"
msgstr "Installiere %s"

#: umc/python/appcenter/__init__.py:844 umc/python/appcenter/__init__.py:903
#, python-format
msgid "Invalid scheme, use http or https: %(base)r/%(key)r: %(value)r"
msgstr ""
"Ungültiges Schema, bitte http oder https benutzen:  %(base)r/%(key)r: "
"%(value)r"

#: umc/python/appcenter/__init__.py:397 umc/python/appcenter/__init__.py:447
msgid ""
"Make sure docker.io is installed, try starting the service with \"service "
"docker start\"."
msgstr ""
"Stellen Sie sicher, dass docker.io installiert ist. Starten Sie den Dienst "
"mit \"service docker start\"."

#: umc/python/appcenter/sanitizers.py:65
msgid "Maybe the machine password changed during the session."
msgstr ""
"Möglicherweise hat sich das Maschinenpasswort während der Sitzung geändert."

#: umc/python/appcenter/sanitizers.py:61
msgid "Maybe your password changed during the session."
msgstr "Möglicherweise hat sich Ihr Passwort während der Sitzung geändert."

#: umc/python/appcenter/__init__.py:93
msgid "Package not found in repository"
msgstr "Paket wurde in keinem Repository gefunden"

#: umc/python/appcenter/sanitizers.py:57
msgid "Please check the join state of the machine."
msgstr "Bitte überprüfen Sie den Status des Domänenbeitritts."

#: umc/python/appcenter/__init__.py:417
msgid ""
"Please either configure a different network for the docker bridge by setting "
"the UCR variable docker/daemon/default/opts/bip to a different network and "
"restart the system,"
msgstr ""
"Bitte konfigurieren Sie ein anderes Netzwerk für die Docker Bridge durch die "
"UCR Variable docker/daemon/default/opts/bip und ein Neustart des Systems,"

#: umc/python/appcenter/__init__.py:233
#, python-brace-format
msgid "Problems connecting to {0} ({1}). Please update {0}!"
msgstr ""

#: umc/python/appcenter/__init__.py:235
msgid "Problems connecting to {} ({})."
msgstr ""

#: umc/python/appcenter/__init__.py:236
msgid ""
"The App Center version of the this host ({}) is not compatible with the "
"version of {} ({})"
msgstr ""

#: umc/python/appcenter/sanitizers.py:61 umc/python/appcenter/sanitizers.py:65
msgid "The credentials provided were not accepted."
msgstr "Die Zugangsdaten wurden nicht akzeptiert."

#: umc/python/appcenter/__init__.py:397
msgid ""
"The docker service is not running! The App Center will not work properly."
msgstr ""
"Der Docker-Dienst läuft nicht! Das App Center wird nicht richtig "
"funktionieren."

#: umc/python/appcenter/sanitizers.py:57
msgid "The server seems to be lacking a proper password file."
msgstr "Dem Rechner scheint eine korrekte Passwortdatei zu fehlen."

#: umc/python/appcenter/sanitizers.py:102
msgid "There already is a component with this name"
msgstr "Es gibt bereits eine Komponente mit diesem Namen"

#: umc/python/appcenter/sanitizers.py:61 umc/python/appcenter/sanitizers.py:65
msgid "This may be solved by simply logging out and in again."
msgstr ""
"Das Problem kann vielleicht bereits durch ein Aus- und erneutes Einloggen "
"gelöst werden."

#: umc/python/appcenter/__init__.py:447
msgid "Unable to start the docker service!"
msgstr "Der Docker-Dienst konnte nicht gestartet werden!"

#: umc/python/appcenter/__init__.py:440
msgid "Unexpected suggestions data."
msgstr "Unerwartete Daten für App-Vorschläge."

#: umc/python/appcenter/__init__.py:337
#, python-format
msgid "Uninstalling %s"
msgstr "Deinstalliere %s"

#: umc/python/appcenter/__init__.py:339
#, python-format
msgid "Upgrading %s"
msgstr "Aktualisiere %s"

#: umc/python/appcenter/__init__.py:748
msgid "installed"
msgstr "installiert"

#: umc/python/appcenter/__init__.py:750
msgid "not installed"
msgstr "nicht installiert"

#: umc/python/appcenter/__init__.py:418
msgid ""
"or disable the docker support in the AppCenter by setting appcenter/docker "
"to false."
msgstr ""
"oder deaktivieren Sie die Unterstützung für Docker im AppCenter indem die "
"UCR Variable appcenter/docker auf false gesetzt wird."

#: umc/python/appcenter/__init__.py:746
msgid "upgradable"
msgstr "aktualisierbar"
